.ci.yaml

```diff
@@ -394,6 +417,29 @@ targets:
           "CHANNEL": "stable"
         }
 
+  - name: Linux_android android_build_all_packages_legacy stable
+    # When making this `bringup: false`, update the task above to
+    # use target_file: android_build_all_packages_jdk17.yaml and
+    # remove its 'dependencies' section.
+    bringup: true # https://github.com/flutter/packages/pull/7099
+    recipe: packages/packages
+    timeout: 30
+    properties:
+      add_recipes_cq: "true"
```

Comment:
I'm going to remove this from the PR; we were causing problems previously where our flake was messing up the recipes CQ because we'd put so many similar tests in it.